src/Worker/ShopwareConsoleCaller.php

```diff
@@ -151,7 +152,34 @@ public function hasError()
     public function resetOutput()
     {
         $this->output = '';
-        $this->returnCode = '';
+        $this->returnCode = -1;
         return $this;
     }
+
+    /**
+     * @param resource $process     process which should be checked for exit code
+     * @param int      $maxWaitTime max time to wait for a running process (in microseconds)
```

Comment:
I'm not sure if I understand that correctly. But if you really mean microseconds, do you think that 5000 is a good default value? That would be only 5 milliseconds...

Reply:
Yeah, mixed up micro and milli ... will correct this

Reply:
But i think, 5 seconds are enough?

Reply:
Not sure. With empty cache Shopware takes sometimes longer tahn 5 seconds...